pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -1104,15 +1104,8 @@ func doActivate(ctx *domainContext, config types.DomainConfig,
 	}
 
 	// Pre-flight checks for containers
-	if config.IsContainer {
-		if len(status.DiskStatusList) == 0 || status.DiskStatusList[0].Format != zconfig.Format_CONTAINER {
-			err := fmt.Errorf("doActivate failed: containers must have the first volume of type OCI")
-			log.Error(err.Error())
-			status.SetErrorNow(err.Error())
-			return
-		} else {
-			status.DiskStatusList[0].MountDir = "/"
-		}
+	if status.IsContainer {
+		status.DiskStatusList[0].MountDir = "/"
```

Comment:
Did you add a check+Fatal somewhere to verify we have at least one element in DiskStatusList?

Reply:
yeah -- that's going into the other PR that's outstanding (was waiting for Avi's PR to re-re-merge ;-))

Reply:
Ok, with me killing that other PR in favor of this one being rebased as a whole -- I added the log.Fatal here for now. Please take a look @eriknordmark